transports/sftp/src/main/resources/META-INF/mule-sftp.xsd

```diff
@@ -252,6 +252,21 @@
                 </xsd:documentation>
             </xsd:annotation>
         </xsd:attribute>
+        <xsd:attribute name="comparator" type="mule:substitutableClass">
```

Comment:
I agree, but as implementation is completely different, I'd prefer to handle FTP in a ticket and a pull-request on it's own